### ISSUE TITLE
Describe > One Variable > Graph

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#3837]
This is working quite well.

My disappointment is the default boring grey of the graphs.  If they are to be a single colour, then couldn't we have a better one?  (I prefer the blue of our Frequency graphs.)  Or could we choose the colour?  
Could you please discuss how we could link these simple dialogues with more  of the flexibility of our ggplot graphing system.  This would be good to explore while in Malawi.

I tried with the 6 factors in the hh Sudan data.  I tried free axis and that was good.  I also tried flipping, and that was good too.  Of course I would also like my "angle" of the x-axis labels, at leasy 90 degrees.  That might just be for the bar charts of course.

**Comment by dannyparsons** [comment]
We could have a different default theme for these graphs. But then should we the same default theme for all graphs? Or can we consider the one/two/three variable graphs differently.

**Comment by maxwellfundi** [comment]
I would quite like it better if we had a sort of uniform theme for all graph. but we could have the one/two/three variable have a different theme. if we were to have blue bars for one var, then we should have those for the two/three and any graph that does not have colour aesthetic.

**Comment by maxwellfundi** [comment]
@rdstern  @dannyparsons I wonder what we agreed for this, can we have a discussion about this?

**Comment by dannyparsons** [comment]
Yes, let's discuss this and think how it would affect other dialogs as it could become a big task.
